src/corehaptics.cs

```diff
@@ -373,6 +377,8 @@ partial interface CHHapticPatternDefinition {
 		NSObject WeakParameterCurve { get; set; }
 		[Export ("ParameterCurveControlPointsKey")]
 		NSObject WeakParameterCurveControlPoints { get; set; }
+		[Advice ("The default value is true.")]
+		bool EventWaveformUseVolumeEnvelope { get; set; }
```

Comment:
hmm... is this only used as an output ?
because if it's an output then, when created, the value will be `false`
This can be bound as `bool?` if the default can (or should) be ignore

Reply:
You need to copy the availability `[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]`
since this is the public API developers will be using (they keys being `internal`)